plugins/generic/usageEvent/UsageEventPlugin.inc.php

```diff
@@ -286,6 +286,18 @@ function _buildUsageEvent($hookName, $args) {
 			$request, null, $canonicalUrlPage, $canonicalUrlOp, $canonicalUrlParams
 		);
 
+		// Make sure we log the server name and not aliases.
+		$configBaseUrl = Config::getVar('general', 'base_url');
+		$requestBaseUrl = $request->getBaseUrl();
+		if ($requestBaseUrl !== $configBaseUrl) {
+			// Make sure it's not an url override (no alias on that case).
+			if (!in_array($requestBaseUrl, Config::getContextBaseUrls()) && 
+					$requestBaseUrl !== Config::getVar('general', 'base_url[index]')) {
```

Comment:
I think I finally got it Clinton... I will just wait for the call to finish and will change that. Thanks... :)